packages/cbioportal-frontend-commons/package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "cbioportal-frontend-commons",
   "description": "cBioPortal Frontend Modules",
-  "version": "0.5.66",
+  "version": "0.5.67-beta.0",
```

Comment:
let's revert this change. publish bot will update this to `0.5.67` and release that version, not the beta version anyway.